packages/components/src/text-input/stories/index.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * External dependencies
+ */
+import NumberFormat from 'react-number-format';
```

Comment:
I've added this to `package.json` 

Is there an alternative or are we okay with adding more dependencies here?

Reply:
This one is actually bigger than the other: https://bundlephobia.com/package/react-number-format@4.6.3

If it's just for the story it's probably fine.